packages/examples/src/fastify.tsx

```diff
@@ -58,7 +58,7 @@ fastify.get('/stream-sample', async (request, reply) => {
     );
   }
   const responseStream = toTextStream(<DescribeCharacter />, new PinoLogger(pinoStdoutLogger));
-  await sendReadableStreamToFastifyReply(reply, responseStream);
+  await sendReadableStreamToFastifyReply(reply, responseStream as any);
```

Comment:
Why did this become necessary?

Reply:
No idea, but whatever caused this to fail locally for me no longer does so it's now removed.